File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Source.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Source {

    @SerializedName("DataType")
    @Expose
    public String dataType;
    @SerializedName("Source")
    @Expose
    public String source;
    @SerializedName("SourceId")
    @Expose
    public Integer sourceId;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Source.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Source (
    @SerializedName("DataType")
    @Expose
    var dataType: String?,
    @SerializedName("Source")
    @Expose
    var source: String?,
    @SerializedName("SourceId")
    @Expose
    var sourceId: Int?
    )